File: C# code/VIATalks/Shared/Room.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataClasses
{
    [Serializable]
    public class Room
    {
        public int Id { get; set; }
        public int RoomNumber { get; set; }
        public char Block { get; set; }
        public int Capacity { get; set; }
        public double Area { get; set; }
        public Room(
            int id,
            int roomNumber,
            char block,
            int capacity,
            double area)
        {
            Id = id;
            RoomNumber = roomNumber;
            Block = block;
            Capacity = capacity;
            Area = area;
        }
    }
}

File: Java code/src/main/java/VIA/VIATalks/Database/data/Room.java
package VIA.VIATalks.Database.data;

import java.io.Serializable;

public class Room implements Serializable {

    // Data to store
    private int id;
    private int roomNumber;
    private char block;
    private int capacity;
    private double area;


    // Constructor
    public Room(int id, int roomNumber, char block, int capacity, double area) {
        this.id = id;
        this.roomNumber = roomNumber;
        this.block = block;
        this.capacity = capacity;
        this.area = area;
    }

    //Getters and Setters
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getRoomNumber() {
        return roomNumber;
    }

    public void setRoomNumber(int roomNumber) {
        this.roomNumber = roomNumber;
    }

    public char getBlock() {
        return block;
    }

    public void setBlock(char block) {
        this.block = block;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public double getArea() {
        return area;
    }

    public void setArea(double area) {
        this.area = area;
    }
}
